packages/client/src/hooks/useNFTs.js

```diff
@@ -87,7 +126,7 @@ export default function useNFTs() {
       type: "SET_NFTS",
       payload: {
         [treasuryAddr]: {
-          [identifier]: tokens,
+          [identifier]: tokens || {},
```

Comment:
Do we know what can cause the tokens to be undefined?

Reply:
I removed this since it's not needed, tokens will be an empty array in case there are none added